modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar_container.js

```diff
@@ -223,7 +223,12 @@ AUI.add(
 											if (calendarResourceId == instance.get('groupCalendarResourceId') && item.get('permissions').MANAGE_BOOKINGS) {
 												defaultCalendar = item;
 											}
-											else if (calendarResourceId == instance.get('userCalendarResourceId') && defaultCalendar == null) {
+
```

Comment:
Hi @natocesarrego, as you imagined I tried to follow the same logic as that of `CalendarDisplayContext.getDefaultCalendar`. Maybe this type of redundant code could be avoided in the future.